robolectric-resources/src/main/java/org/robolectric/res/ResourceTableFactory.java

```diff
@@ -0,0 +1,47 @@
+package org.robolectric.res;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Modifier;
+
+public class ResourceTableFactory {
+
+  public static ResourceTable newResourceTable(String packageName, ResourcePath... resourcePaths) {
```

Comment:
Agreed, it would be easy to extract an interface but we can decide on how we would make that pluggable later?